apps/editor/index.d.ts

```diff
@@ -570,8 +570,19 @@ declare namespace toastui {
     public styleToSelectedCells(onStyle: SquireExt, options?: object): void;
   }
 
+  interface MarkdownRange {
+    from: CodeMirror.Position;
+    to: CodeMirror.Position;
+    ollapsed: boolean;
+  }
+
+  class CodeMirrorExt {
+    getCurrentRange(): MarkdownRange;
+    getEditor(): CodeMirrorType;
+  }
+
   // @TODO: change toastMark type definition to @toast-ui/toastmark type file through importing
-  class MarkdownEditor {
+  class MarkdownEditor extends CodeMirrorExt {
```

Comment:
Why did you add the `CodeMirrorExt` type to handle the event issue?

Reply:
This one is actually a type bug fix. MarkdownEditor extends a CodeMirrorExt class:

https://github.com/nhn/tui.editor/blob/master/apps/editor/src/js/markdownEditor.js#L113

but this isn't represented in the types. I added two methods from CodeMirrorExt I found useful when developing custom commands: ideally I think they would all be added but this is quite a lot more work!